include/Utils.h

```diff
@@ -23,6 +23,8 @@ std::string variable(const Value *Val);
  */
 std::string address(const Value *Val);
 
+bool isNumber(const std::string& s);
```

Comment:
Is this not in the standard library? I would think you could just call some version of `atoi`?

Reply:
I don't think there is; at most there is isdigit but nothing for an entire string

Reply:
(You'd check its return value to determine if it succeeded, since it is a conversion function.)

Reply:
This conversation still hasn't been resolved, and should be before we merge.

Reply:
So even if we use `strtol`, it wouldn't be a simple, short one-liner, hence why I still think defining a utility `isNumber` function is a good choice here. Since `isNumber` currently look likes: 
`char* endPtr;
  std::strtol(s.c_str(), &endPtr, 10);
  return endPtr != s.c_str() && *endPtr == '\0';`
  I'd rather have it remain a utility function with a descriptive name

Reply:
Alright, thanks for looking into it. I agree that keeping a descriptive name is a worthwhile goal.